modules/common/development/scripts/sysbench_fileio_test.nix

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{pkgs, ...}:
+with pkgs;
+  pkgs.writeScriptBin "sysbench-fileio-test" ''
+
+    # Test set to be run with sysbench
+
+    # These variables need to be given on the command line. For example: ./fileio_test 20
+    THREADS="''$1"
+
+    # Default expected power mode (on Orin targets)
+    ExpectedPowerModeNo=3
+
+    # Create a directory for the results
+    RESULT_DIR="sysbench_results"
+    echo -e "\nCreating directory for test results:\n./''$RESULT_DIR"
+    mkdir -p ''$RESULT_DIR
+
+    # Create test_info file with system information
+    echo -e "\nSaving information about test environment to ./''$RESULT_DIR/test_info\n"
+    echo -e "''$(lscpu)" "\n\n" "''$(free)" "\n\n" "''$(df)" "\n\n" >> ./''$RESULT_DIR/test_info
+    echo -e "\nHost: ''$(hostname)\n" | tee -a ./''$RESULT_DIR/test_info
+
+    # Check cpu architecture for specific actions
+    if lscpu | grep -qw "aarch64"; then
+        echo -e "\nSystem architecture is aarch64." | tee -a ./''$RESULT_DIR/test_info
+        if hostname | grep -qw "ghaf-host"; then
+            echo -e "''$(nvpmodel -q)" "\n" >> ./''$RESULT_DIR/test_info
```

Comment:
I want to point out nvpmodel is NVIDIA specific, and not aarch64 specific

Reply:
More compatible way would be probably to just enable the cores, and set the cores to use performance governor through sysfs. (That is actually partly what nvpmodel internally does.) Maybe we could have both?

Reply:
> I want to point out nvpmodel is NVIDIA specific, and not aarch64 specific

In our context we will check nvpmodel only on Orin AGX/NX targets. I cleaned and separated the check script to own file.

Reply:
This is good and more future proof. Now if we start testing the iMX8 targets, it won't cause problems